### ISSUE TITLE
Female recruits in Group V

### ISSUE THREAD

**Issue by rnyoakum** [raceintospace/raceintospace#344]
In the discussion of issue https://github.com/raceintospace/raceintospace/issues/329#issuecomment-609073823, I asked

> By the way, it would be nice if we had female candidates in Groups IV and V. Sadly, I'm guessing there just aren't any historical options who were considered during that time period?

@peyre replied (https://github.com/raceintospace/raceintospace/issues/329#issuecomment-609080165)

> Yes, that's my understanding, based on my discussions with Fritz and Michael. Groups I-III used members of the Mercury 13 and the Soviet Female Group. After then, no women were recruited into either space program until 1978. The youngest of the Mercury 13 would be kind of a stretch to place in Group IV, and I don't know who we'd use for the USSR.
> 
> For Group V we could, of course, include some of those from 1978 (Sally Ride, Anna Fisher, Shannon Lucid, Judith Resnik, Reha Seddon, Kathryn Sulliavn) and the 1980 Soviet recruitment (Svetlana Savitskaya, Yekaterina Ivanova, Natalya Kuleshova, Irina Pronina, Irina Latysheva, Galina Vasiliyevna, Yelena Ivanovna, Larisa Grigoriyevna, Tamara Sergeyevna).

While it sounds like it would break the historical immersion too much to fit female candidates into Group IV (RIS uses real candidates, and I wouldn't want to change that), it would be nice to add some female candidates to group V. It wouldn't require replacing any of the current candidates; adding new entries to the roster would be reasonably simple.

Obvious issues:

- Candidates would have to be the appropriate age for roughly the entire span when Group V is available (1969-1977, by my reckoning)
- How to determine appropriate skill levels for the candidates

**Comment by peyre** [comment]
That's not a bad idea.  We _could_ maybe use the youngest three of the Mercury 13 for Group IV; there are enough of the group to fill 12 slots of course.  For Group V, I'd suggest Sally Ride of course.  For the other two, maybe Shannon Lucid and Margaret Seddon; they're the oldest. 

For the Soviets, it's gotta be Svetlana Savitskaya and Yekaterina Ivanova; for the other, it's kind of tough, since none of them flew in space.  Dobrokvashina and Pozharskaya are the oldest of the others, but how to tell between them?  They were selected, and retired, at the same time.  Maybe Dobrokvashina, since she trained for a Soyuz female mission that was later canceled, and Pozharskaya doesn't show anything like that in her profile (http://www.spacefacts.de/bios/cosmonauts/english/pozharskaya_larisa.htm).

**Comment by peyre** [comment]
Just found a Smithsonian Mag article that mentions some candidates I hadn't heard of before.  Marina Popovich was a pilot who later became the USSR's first woman military test pilot.  She was turned down for health reasons, but perhaps relatively relaxed standards several years later would let her in.  Tatyana Morozycheva was a strong candidate but wasn't selected.  I could track down a contemporary military pilot to fill the other slot in Soviet Group IV.
[https://www.smithsonianmag.com/science-nature/first-group-female-cosmonauts-trained-conquer-final-frontier-180971900/](url)

**Comment by peyre** [comment]
So, looking over the Mercury 13, the youngest (Stumbough, Gorelick, and Funk) are currently in Group III and would be moved to Group IV.  The next youngest are Cobb, Sloan, and Hurrle.  I'm comfortable with Cobb remaining in Group I since she was the first to get involved in the testing, so if she stays there, the next youngest would be Leverton.  Hart should probably be moved up from Group II to I since she was the oldest in the group, and could be replaced by Hixson.  That would give us:
**GROUP I**
Hart
Cagle
Cobb
**GROUP II**
J.Dietrich
M.Dietrich
Hixson
**GROUP III**
Sloan
Hurrle
Leverton
**GROUP IV**
Stumbough
Gorelick
Funk
**GROUP V**
Ride
Lucid
Seddon

**Comment by peyre** [comment]
About the skills: this is the distribution of the 'naut roster (which I haven't changed from the original): 
`0    117  11%
1    468  44%
2    346  32%
3    113  11%
4     23   2%
Tot 1067`
I'll leave the skills for Groups I-III in place (but have, say, Hart and Hurrle/Hixson's skills follow them).  To generate new skills, I'll work out 3 (very good) and 4 (excellent) where I think I can for CA, EV, and EN and assign a few at random for LM & DO, then for everything else, assign skills at random, weighted as follows:
`0    117  12.5%   1
1    468  50%    2-6
2    346  37%    7-0
Tot  931`
See, I knew I still had a use for my old d10s from my D&D days!

**Comment by hweimer** [comment]
In 1969, Sally Ride and Margaret Seddon were 18 and 21, respectively, so adding them might be a bit of a stretch. Shannon Lucid is fine at age 26, as would be the youngest of the Mercury 13 (Wally Funk) who was 38 in 1977. Moving Funk into group V would free a slot for the last of the Mercury 13 (Rhea Woltman), but she would have to go into one of the earlier groups.

**Comment by peyre** [comment]
It's unfortunate!  Such a stretch to get historical women into these Group IV and V slots.

Actually I'm not too worried about 1969 as a cutoff date.  If a player is down to recruiting Group V, they're probably into the 70s at that point.  Ride would have been 23 in 1974, not beyond the realm of possibility.  Still, that's awfully young, and historically, she was recruited for NASA fresh out of college - so you're quite right that our time period is really too early for her.

You make a good point there!  I keep thinking about what a stretch it is including the Mercury 13 in Group IV, but Funk would've been only 30 in 1969, or 35 in 1974.  Not beyond the realm of the believable, especially given that Dyomin flew at 48.  So well, I could reshuffle these folks and put Funk into Group V.

Are we planning to convert the 'naut roster to a plaintext file any time soon?  That would make it easy to do this.

**Comment by hweimer** [comment]
To put it in another way, Ride and Seddon would be way younger than any of the current group V members, by 11 and 7 years, respectively (currently, the youngest is Brian O'Leary, born in 1940). On the other hand, some of the group V members are also a bit of a stretch in the other direction, especially Milt Thompson, who was born in 1926.

Moving the roster files to JSON can be done rather easily once #478 is merged, which is blocked by #416.

**Comment by peyre** [comment]
Just so you know, I've known about #478 for some time, but I don't really understand what it's about so I hadn't known what to do about it - just as well, it sounds like.

**Comment by peyre** [comment]
Ok, so the roster so far is:

**GROUP I**
Hart
Cagle
Cobb
**GROUP II**
J.Dietrich
M.Dietrich
Hixson
**GROUP III**
Sloan
Steadman
Leverton
**GROUP IV**
Stumbough
Gorelick
Hurrle
**GROUP V**
Funk
Lucid
Seddon

And for the Soviets:
**GROUP IV**
Popovich
Morozycheva
Pronina
**GROUP V**
Savitskaya
Ivanova
Dobrokvashina

**Comment by hweimer** [comment]
I would swap Hurrle and Sloan, otherwise the list looks fine to me.

One other thing (which maybe should be discussed in a separate issue): Wouldn't it make sense to enable female recruits at some point (say, 1972) even when you didn't get the newscast? Right now, the game is (statistically) treating women even worse than real life because most games do no feature any female nauts at all.

**Comment by peyre** [comment]
I hadn't thought about that, allowing women sometime in the 70s.  I don't see a problem with it.  Of course, it's helpful that the game now allows you to enable recruiting women via the config file.

Oh yes, I see Sloan is younger than Hurrle - I must've gotten them backward.

**Comment by hweimer** [comment]
Maybe it would be even better to raise the probability for the newscast. Historically, the Soviet side got the newscast in 1962, while on the U.S. side it is reasonable to assume the year 1972, when the Equal Employment Opportunity Act was passed. To make some crude statistics, that's an average timespan of 10 years starting from 1957, meaning you should have a 5% chance per turn to see the newscast. Unfortunately, that's not something that is straightforward to implement given how the event selection code works.

Oh, and forget everything about what I said concerning Margaret Seddon being too young at age 21. I wasn't aware of Tatyana Kuznetsova being selected at age 20, so there is historical precedent.

**Comment by rnyoakum** [comment]
> Maybe it would be even better to raise the probability for the newscast.

I agree that it would be hard to do this, but it would be feasible to do a pass through the event list after they're generated and replace a timely event with the Female Astronauts/Cosmonauts event. Yet I doubt if that's an appropriate choice because the event **requires** a female astronaut/cosmonaut to be chosen. I _would_ suggest doing the above with a new event that enables, but does not require, selection of a female astronaut/cosmonaut - if that didn't entail so many of its own problems.

Another option would be to silently enable female astronauts after a certain period of time, regardless of events. The news event requires recruitment, so it would still have a distinct effect. This approach would silently change the rules of the game, however, which is normally a Bad Idea.

**Comment by peyre** [comment]
Or, possibly, we could create an additional news item that allows recruiting women, in addition to the one that requires it - that would double the chances of getting females.

**Comment by hweimer** [comment]
Adding a separate news event for allowing but not requiring to select female nauts runs against the current logic of the naut selection code. However, this also seems to imply that once you get the news event you will have to select at least one female candidate in all subsequent groups as well? At least that's not what the text of the event says as this only mentions the next group.

Anyway, although closing this issue is required for implementing such a change, this is best discussed in a separate issue.

**Comment by peyre** [comment]
Very tiny side note: I think I would raise Hart's LM skill to 3 since (as far as I know) she was the only one of the Mercury 13 to become a helicopter pilot.

**Comment by peyre** [comment]
I watched Netflix's "Mercury 13" last night.  It was largely review for me, but neat to see clips of things I'd read about before.

**Comment by peyre** [comment]
Ok, I've done this one!  It seems to work perfectly.  If I got something wrong here, please let me know.
